hazelcast/src/main/java/com/hazelcast/map/impl/operation/EvictAllOperation.java

```diff
@@ -46,10 +46,6 @@ public EvictAllOperation(String name) {
 
     @Override
     public void run() throws Exception {
-
-        // TODO this also clears locked keys from near cache which should be preserved.
-        clearLocalNearCache();
```

Comment:
I couldn't find a test for this scenario, so I wrote one: https://github.com/hazelcast/hazelcast/pull/8869/files#diff-1255ed0d931aed11a9aa83ddb3864b8fR799

:+1: